src/views/landing.njk

```diff
@@ -42,7 +36,7 @@
       <p>
         <a href="https://www.gov.uk/government/publications/late-filing-penalties/late-filing-penalties#appealing" target="_blank">Read the guidance on appealing a penalty.</a>
       </p>
-      <h3 class="govuk-heading-s">You'll need:</h3>
+      <h3 class="govuk-heading-s">You’ll need:</h3>
```

Comment:
Dedicated apostrophe char renders bit nicer UI and it does not break template syntax highlight. I suggest we use it in cases like that instead of using single quote.

How to achieve it? It is `option+shift+]` on Mac.